vassal-app/src/main/java/VASSAL/build/GameModule.java

```diff
@@ -438,9 +438,16 @@ public void fireKeyStroke(KeyStroke stroke) {
   }
 
   public void fireKeyStroke(NamedKeyStroke stroke) {
-    if (stroke != null && !stroke.isNull()) {
-      fireKeyStroke(stroke.getKeyStroke());
+    if (stroke == null || stroke.isNull() || stroke.getKeyStroke() == null) {
+      return;
     }
```

Comment:
Aren't you calling `stroke.getKeyStroke()` and checking it's nonnull twice in the case where it's nonnull?

Reply:
Whoops, of course.

Fixed, rebased, squashed and force-pushed.